### PR TITLE
(docs) Basic Logback docs

### PR DESCRIPTION
**Pull request by gguillotte**
This was done in parallel to PR #874 and should be better integrated with its work. I'm making this PR for preliminary review.

### CODE REVIEW

File: documentation/config_file_logbackxml.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+### Settings
+
+#### `level`
+
+To modify Puppet Server's logging level, change the `level` attribute of the `root` tag. For instance, the default logging level is set to `info`:
+
+    <root level="info">
+
+To enable debug logging, change `info` to `debug`:
```
Comment (fpringvaldsen) [inline documentation/config_file_logbackxml.md]: Might also be worth nothing the `error` and `trace` levels

File: documentation/config_file_logbackxml.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Debug-level logging includes Puppet Server profiling data.
+
+#### Logging location
+
+You can change the file to which Puppet Server writes its logs in the `appender` section named `F1`. For instance, the default location is set to `/var/log/puppetlabs/puppetserver/puppetserver.log`:
+
+~~~ xml
+...
+    <appender name="F1" class="ch.qos.logback.core.FileAppender">
+        <!-- TODO: this path should not be hard-coded -->
```
Comment (fpringvaldsen) [inline documentation/config_file_logbackxml.md]: Should we leave this TODO comment out of our examples?

**Comment by fpringvaldsen** [comment]
This is looking good! I only had two small pieces of feedback

File: documentation/config_file_logbackxml.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: logback.xml"
+canonical: "/puppetserver/latest/config_file_logbackxml.html"
+---
+
+Puppet Server’s logging is routed through the Java Virtual Machine's (JVM) [Logback library](http://logback.qos.ch/). By default, it logs to `/var/log/puppetlabs/puppetserver/puppetserver-access.log`, the log level is ‘INFO’, and Puppet Server sends nothing to `syslog`.
```
Comment (cprice404) [inline documentation/config_file_logbackxml.md]: The "main" logging actually defaults to `/var/log/puppetlabs/puppetserver/puppetserver.log`.  Inside of the `webserver` section of the `webserver.conf` file, though, you can optionally specify a configuration for HTTP request logging, e.g.:

https://github.com/puppetlabs/puppet-server/blob/206eb64b302b9ccd0db53b6ef247f86cc0a90fa4/ezbake/config/conf.d/webserver.conf#L2

That points to a *different* logback config file which controls the HTTP request logging, and by default we log the HTTP request info to `/var/log/puppetlabs/puppetserver/puppetserver-access.log`.

Comment (cprice404) [inline documentation/config_file_logbackxml.md]: Oh, I see, you cover the HTTP request logging below.  So it's probably just a typo on the file path here then.

**Comment by cprice404** [comment]
LGTM other than the one file path typo.  @gguillotte did you want us to go ahead and merge this soon-ish (pending minor corrections above), or just keep it in the hopper here until you decide what your longer-term plans are?

**Comment by gguillotte** [comment]
@cprice404 @fpringvaldsen I'll add a few links from it to the awesome advanced doc you wrote up, get a docs review on the whole thing, and then we can revisit and merge. Thanks for the comments!

**Comment by gguillotte** [comment]
Updated with feedback from @fpringvaldsen and @cprice404, and added links to the `config_logging_advanced` doc.

**Comment by gguillotte** [comment]
Pinging @fpringvaldsen @cprice404 -- if it wasn't clear, this is ready to merge as far as I'm concerned.

**Pull request merged**